File: mifosng-android/src/main/java/com/mifos/objects/client/ChargesRename.java
package com.mifos.objects.client;

/**
 * Created by nellyk on 2/15/2016.
 */

/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

public class ChargesRename {

    private Integer chargeId;
    private String amount;
    private String locale;
    private String dueDate;


    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getDueDate() {
        return dueDate;
    }

    public void setDueDate(String dueDate) {
        this.dueDate = dueDate;
    }


    public Integer getChargeId() {
        return chargeId;
    }

    public void setChargeId(Integer chargeId) {
        this.chargeId = chargeId;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }
}

File: core/datastore/src/main/java/com/mifos/core/objects/client/ChargesRename.kt
package com.mifos.core.objects.client

/**
 * Created by nellyk on 2/15/2016.
 */
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
data class ChargesRename (
    var chargeId: Int? = null,

    var amount: String? = null,

    var locale: String? = null,

    var dueDate: String? = null
)